pages/pipelines/command_step.md.erb

```diff
@@ -89,6 +89,14 @@ _Optional attributes:_
       <em>Example:</em> <code>RAILS_ENV: "test"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>id</code></td>
+    <td>
+      A string to identify the step. It is exported as the `BUILDKITE_STEP_IDENTIFIER` environment variable.<br>
```

Comment:
Awesome! Do we use “export” anywhere in here? It means something specific to Linux environment variables, but perhaps that’s not what’s happening here.

What about something like…

```suggestion
      A string to identify the step. The value is available in the `BUILDKITE_STEP_IDENTIFIER` <a href="/docs/pipelines/environment-variables">environment variable</a><br>
```